index.html

```diff
@@ -13,12 +13,17 @@
         <span><a class="logo" href="#">Mudasiri</a></span>
         <nav>
             <i class="fa-solid fa-bars"></i>
+            <ul class="nav-items">
+                <li>Portfolio</li>
+                <li>About</li>
+                <li>Contact</li>
+            </ul>
         </nav>
     </header>
     <section class="intro">
```

Comment:
- [ ] Well done, you made great effort to make the changes as requested by the previous reviewer. But you still a few things to do, please confirm the proper order of the project as given in the Figma design. I cannot see that UBER project section any longer. THe order:
- [ ] Tonic
- [ ] Multipost
- [ ] facebook 
- [ ] uber